README.md

```diff
@@ -82,6 +83,17 @@ docker run -p 8000:8000 wagtailcms
 
 ```
 
+## Environment variables
+
+| Variable        | Default       | Description                       |
+| --------------- | :-----------: | --------------------------        |    
+| DBHOST         | unistatsdb.postgres.database.azure.com            | DB host url/string                |
+| DBPORT         | 5432          | DB connection port                |
+| DBNAME         | unistatsdevdb        | db name to use                    |
```

Comment:
Sentence case description for `DBNAME` to be consistent with other rows (variables)